trillian.proto

```diff
@@ -139,6 +139,10 @@ message Tree {
   // The public key used for verifying tree heads and entry timestamps.
   // Readonly.
   PublicKey public_key = 14;
+
+  // Interval after which a new signed root is produced even if there have been
+  // no submission.  If zero, this behavior is disabled.
+  int64 force_new_root_millis = 15;
```

Comment:
Duration is standard for Google APIs. I'd strongly suggest we keep to it. (On a similar case, we should be using `google.protobuf.Timestamp` where appropriate as well.)

There are already conversion functions between `durationpb.Duration` and `time.Duration` under [ptypes](https://godoc.org/github.com/golang/protobuf/ptypes#Duration).

(FWIW, personally, I find the Seconds/Nanos representation a bit strange too.)